.github/workflows/bid-probe.yml

```
@@ -0,0 +1,137 @@
+# in-cluster probe remains the primary bid-health signal. The schedule is added
+# only after manual dispatches prove the pinning works and leak zero orders, and
+# the in-cluster probe is not retired until the two have agreed for weeks.
+
+on:
+  workflow_dispatch:
+    inputs:
+      cluster:
+        description: "Limit to one cluster (blank = all three)"
+        type: choice
+        default: ""
+        options: ["", "alphavps", "onidc", "hetzner_hel"]
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)